js/ui/paygUnlockDialog.js

```diff
@@ -328,6 +333,53 @@ var PaygUnlockDialog = new Lang.Class({
         }
     },
 
+    _onSuccess: function(mainBox) {
```

Comment:
I don't think this is the right name for this function. It's called when the expiry time changes. We hope that that will only happen because more time is added, but it's possible that it could be emitted without any change, or with a negative change.